web/src/main/webapp/includes/global_functions_javascript.js

```diff
@@ -1,3 +1,21 @@
+  function checkDataStatus() {
+
+        objImage=document.getElementById('status_top');
+        if (objImage != null && objImage.src.indexOf('images/icon_UnsavedData.gif')>0) {
+            return confirm('<fmt:message key="you_have_unsaved_data" bundle="${resword}"/>');
+        }
+
+        return true;
+    }
+    function gotoLink() {

```

Comment:
this function is not called from anywhere. Can you look into this?

Reply:
those 2 functions are called in many jsp pages, like initialDataEntryNw.jsp, and also can find same function definitions, but it's in html comments block